Formula/sync_gateway.rb

```diff
@@ -16,10 +15,41 @@ class SyncGateway < Formula
   end
 
   depends_on "go" => :build
+  depends_on :gpg => :build
+
+  resource "depot_tools" do
+    url "https://chromium.googlesource.com/chromium/tools/depot_tools.git",
+      :revision => "935b93fb9bf367510eece7db8ee3e383b101c36d"
+  end
 
   def install
-    system "make", "buildit"
-    bin.install "bin/sync_gateway"
+    ENV["GOPATH"] = buildpath/"godeps"
+    repo_cache = HOMEBREW_CACHE/"repo_cache/#{name}/.repo"
+    repo_cache.mkpath
+
+    inreplace "manifest/default.xml", "git://", "https://" unless build.head?
```

Comment:
Upstream has already "upgraded" the URLs for their go dependencies to use https in HEAD, but as of the 1.3.1 stable tag, they were all still `git://`, so this makes sure we use https with stable, too. It can be removed for the next release.